src/com/projectkorra/projectkorra/airbending/combo/AirSweep.java

```diff
@@ -62,6 +63,7 @@ public AirSweep(final Player player) {
 		this.speed = getConfig().getDouble("Abilities.Air.AirSweep.Speed");
 		this.knockback = getConfig().getDouble("Abilities.Air.AirSweep.Knockback");
 		this.cooldown = getConfig().getLong("Abilities.Air.AirSweep.Cooldown");
+		this.hitRadius = getConfig().getDouble("Abilities.Air.AirSweep.HitRadius");
```

Comment:
We typically just have this option called 'Radius' in our other moves. Might be worth changing it to that just for consistency's sake.